tools/development/register-boot2docker.sh

```diff
@@ -11,10 +10,14 @@ fi
 # This is just here to make sure your environment is sane
 docker info
 
-CONSOLE_ARGS=""
-if [ -t 1 ]; then
```

Comment:
Ah, was curious about why those were needed. Thanks for clearing up. I can add them back in.